components/mpas-ocean/src/Registry.xml

```diff
@@ -1962,7 +1974,11 @@
             <var name="gotmEpsbTopOfCell"/>
             <var name="gotmDissTopOfCell"/>
             <var name="gotmLengthTopOfCell"/>
-            <var name="ssh_sal"/>
+            <var name="pgf_sal"/>
+            <var name="temp_twd"/>
+            <var name="normalCoeff"/>
+            <var name="tangentialCoeff"/>
+            <var name="topographicWaveDrag"/>
```

Comment:
I don't believe these need to be in the restart stream, since they are calculated in an init routine